src/main/resources/listLookup.json

```diff
@@ -604,8 +602,7 @@
         "welshFriendlyName": "Rhestr Achosion Dyddiol Sifil yn y Llys Sirol yng Nghanol Llundain",
         "shortenedFriendlyName": "County Court at Central London Civil Daily Cause List",
         "url": "county-court-london-civil-daily-cause-list",
-        "jurisdictions": ["Civil"],
-        "welshJurisdictions": ["Llys Sifil"],
+        "jurisdictionTypes": ["Civil"],
```

Comment:
This should be 'Civil Court' instead of 'Civil'. Same comment applies to the ones below